utilities/aap.py

```
@@ -0,0 +1,619 @@
+
+def teardown_awx() -> None:
+    """Remove AWX deployment via Helm and delete namespace."""
+    LOGGER.info("Tearing down AWX deployment")
+    try:
+        _run_shell(f"helm uninstall {AWX_HELM_RELEASE_NAME} --namespace {AWX_NAMESPACE}")
+    except subprocess.CalledProcessError:
+        LOGGER.warning("Helm uninstall failed (may already be removed)")
+    try:
+        _run_shell(f"oc delete namespace {AWX_NAMESPACE} --wait=false")
+    except subprocess.CalledProcessError:
+        LOGGER.warning(f"Failed to delete namespace '{AWX_NAMESPACE}'")
```

Ruff (0.15.20)

[warning] Logging statement uses f-string

(G004)